x/compute/internal/keeper/keeper.go

```diff
@@ -487,7 +487,7 @@ func (k Keeper) Instantiate(ctx sdk.Context, codeID uint64, creator sdk.AccAddre
 }
 
 // Execute executes the contract instance
-func (k Keeper) Execute(ctx sdk.Context, contractAddress sdk.AccAddress, caller sdk.AccAddress, msg []byte, coins sdk.Coins, callbackSig []byte) (*sdk.Result, error) {
+func (k Keeper) Execute(ctx sdk.Context, contractAddress sdk.AccAddress, caller sdk.AccAddress, msg []byte, coins sdk.Coins, callbackSig []byte, ibcHooks bool) (*sdk.Result, error) {
```

Comment:
Not a fan of adding a special parameter for ibcHooks. This bloats the interface even more than it is currently is. Isn't there any other way to do this?